pyomo/core/plugins/transform/strip_bounds.py

```diff
@@ -0,0 +1,68 @@
+"""Transformation to strip variable bounds from a model."""
+import textwrap
+
+from pyomo.core.base.var import Var
+from pyomo.core.plugins.transform.hierarchy import IsomorphicTransformation
+from pyomo.util.plugin import alias
+from pyomo.core.kernel.component_map import ComponentMap
+from six import iteritems
+from pyomo.core.kernel.set_types import Reals
+
+__author__ = "Qi Chen <https://github.com/qtothec>"
+
+
+class VariableBoundStripper(IsomorphicTransformation):
+    """Strips bounds from variables."""
+
+    alias('core.strip_var_bounds',
+          doc=textwrap.fill(textwrap.dedent(__doc__.strip())))
+
+    def __init__(self):
+        """Initialize the transformation."""
+        super(VariableBoundStripper, self).__init__()
+
+    def _apply_to(self, instance, strip_domains=True, tmp=False):
+        """Apply the transformation.
+
+        Args:
+            instance (Block): the block on which to strip variable bounds
+            strip_domains (bool, optional): strip the domain for discrete
+                variables as well
+            tmp (bool, optional): Whether the bound stripping will be
```

Comment:
I'm going to wait on this for resolution of #277, since it would affect some other existing transformations as well.

Reply:
I don't see any need to wait for resolution of #277 (It will take longer than the 5.4 release deadline). If we want this in the 5.4 release, we should proceed with the requested changes as is.